templates/CRM/Event/Form/Participant.tpl

```diff
@@ -107,7 +104,7 @@
           <tr class="crm-participant-form-block-status_id">
             <td class="label">{$form.status_id.label}</td>
             <td>{$form.status_id.html}{if $event_is_test} {ts}(test){/ts}{/if}
-              <div id="notify">{$form.is_notify.html}{$form.is_notify.label}</div>
+              <div id="send_notification">{$form.is_notify.html}{$form.is_notify.label}</div>
```

Comment:
This was working for me as-is before, but I think it's ok to change except I might not change the id, I'd add a class and then target the class in the javascript. Because if somebody is doing something with this for styling or other customization then that will still work.